CMakeLists.txt

```diff
@@ -1,14 +1,30 @@
 ##---------------------------------------------------------------------------##
 ## Cabana/CMakeLists.txt
 ##---------------------------------------------------------------------------##
-cmake_minimum_required(VERSION 3.9)
+cmake_minimum_required(VERSION 3.11)
 
 ##---------------------------------------------------------------------------##
 # Define your project name and set up major project options
 ##---------------------------------------------------------------------------##
 project(Cabana LANGUAGES CXX)
 set(PROJECT_VERSION "0.3.0-dev")
 
+# Download and unpack googletest at configure time
+include(FetchContent)
+FetchContent_Declare(
+  googletest
+  URL https://github.com/google/googletest/archive/release-1.10.0.tar.gz
```

Comment:
The URL download fails on the LLNL machines I've tried where the Git repo has worked fine. Not sure if there's something I can change on my end

Reply:
We can move to `git` but will need to update our Docker image - @dalg24 thoughts?

Reply:
@dalg24 does the jenkins machine have no internet connection?